ts/src/oxfun.ts

```diff
@@ -0,0 +1,2843 @@
+            'liquidationPrice': this.safeNumber (position, 'estLiquidationPrice'),
+            'entryPrice': this.safeNumber (position, 'entryPrice'),
+            'unrealizedPnl': this.safeNumber (position, 'positionPnl'),
+            'realizedPnl': undefined, // todo check
+            'percentage': undefined,
+            'contracts': this.safeNumber (position, 'position'),
+            'contractSize': undefined,
+            'markPrice': this.safeNumber (position, 'markPrice'),
+            'lastPrice': undefined,
+            'side': undefined,
+            'hedged': undefined,
+            'timestamp': undefined,
+            'datetime': undefined,
+            'lastUpdateTimestamp': this.safeInteger (position, 'lastUpdatedAt'),
+            'maintenanceMargin': undefined,
+            'maintenanceMarginPercentage': undefined,
+            'collateral': undefined,
+            'initialMargin': undefined,
+            'initialMarginPercentage': undefined,
+            'leverage': undefined,
+            'marginRatio': undefined,
+            'stopLossPrice': undefined,
+            'takeProfitPrice': undefined,
+        });
+    }
+
+    async createOrder (symbol: string, type: OrderType, side: OrderSide, amount: number, price: Num = undefined, params = {}): Promise<Order> {
+        /**
+         * @method
+         * @name oxfun#createOrder
+         * @description create a trade order
+         * @see https://docs.ox.fun/?json#post-v3-orders-place
+         * @param {string} symbol unified symbol of the market to create an order in
+         * @param {string} type 'market', 'limit', 'STOP_LIMIT' or 'STOP_MARKET'
+         * @param {string} side 'buy' or 'sell'
+         * @param {float} amount how much of currency you want to trade in units of base currency
+         * @param {float} [price] the price at which the order is to be fullfilled, in units of the quote currency, ignored in market orders
+         * @param {object} [params] extra parameters specific to the exchange API endpoint
+         * @param {int} [params.clientOrderId] a unique id for the order
+         * @param {int} [params.timestamp] in milliseconds. If an order reaches the matching engine and the current timestamp exceeds timestamp + recvWindow, then the order will be rejected.
+         * @param {int} [params.recvWindow] in milliseconds. If an order reaches the matching engine and the current timestamp exceeds timestamp + recvWindow, then the order will be rejected. If timestamp is provided without recvWindow, then a default recvWindow of 1000ms is used.
+         * @param {string} [params.responseType] FULL or ACK
+         * @param {float} [params.cost] the quote quantity that can be used as an alternative for the amount for market buy orders
+         * @param {float} [params.triggerPrice] The price at which a trigger order is triggered at
+         * @param {float} [params.limitPrice] Limit price for the STOP_LIMIT order
```

Comment:
@rayBastard @yzh-pelle if the user provides triggerPrice in `params` then the `price` is the price for the stop_limit, we don't need this exchange-specific value parsing

Reply:
Should we delete `params.limitPrice` from the describing?